src/mixins/search-mixin.ts

```diff
@@ -158,7 +158,8 @@ export class SearchMixin extends Mixins(CommonMixin) {
   }
 
   get isChangeNameXpro (): boolean {
-    return XproMapping.CHG.includes(this.getSearchBusiness?.legalType)
```

Comment:
This would have worked for the types that are the same between Namerequest and LEAR.